tests/ir/.gitignore

```diff
@@ -0,0 +1 @@
+/Output
```

Comment:
I think putting tests close to the code they are testing is the most sensible road.